From 29e71752804d4a547518e0e51c6ad8ae58200027 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Alberts=20Muktup=C4=81vels?= <alberts.muktupavels@gmail.com>
Date: Mon, 24 Feb 2014 10:08:17 +0200
Subject: gnome-panel.desktop: remove StartupNotify line

With new glib (2.39.1+) versio gnome-panel failed to register
with session manager allways reaching timeout. This simple commit
fix that bug.

diff --git a/gnome-panel/gnome-panel.desktop.in.in b/gnome-panel/gnome-panel.desktop.in.in
index f90467d..bea8e9a 100644
--- a/gnome-panel/gnome-panel.desktop.in.in
+++ b/gnome-panel/gnome-panel.desktop.in.in
@@ -4,7 +4,6 @@ _Name=Panel
 _Comment=Launch other applications and provide various utilities to manage windows, show the time, etc.
 Icon=gnome-panel
 Exec=gnome-panel
-StartupNotify=true
 Terminal=false
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-panel
-- 
cgit v0.10.1

